kenji_crosland/app/sass/_base.scss

```diff
@@ -0,0 +1,41 @@
+@import url(https://fonts.googleapis.com/css?family=Nunito);
+@import url(https://fonts.googleapis.com/css?family=Dancing+Script);
+@import url(https://fonts.googleapis.com/css?family=Ledger);
+
+html {font-size: 1.125em;}
+
+body {
+  font-family: 'Ledger', serif;
+  background-color: $light-background;
+  font-weight: 400;
+  line-height: 1.45;
+  color: #333;
+}
+
+p {margin-bottom: 1.3em;}
+
+h1, h2, h3, h4 {
```

Comment:
Maybe put these in `%headings {...}` and then extend it for all the headings that use it?  Not necessarily better than what you have, but it would be Sass-ier  :)